File: Verifica/angularapp/src/app/Models/usuario.dto.ts
export class UsuarioDto {
  id?: number;
  nombre?: string;
  apellido1: string;
  apellido2: string;
  username: string;
  password: string;
  guid: string;
  email: string;
  telefono: string;
  admin: boolean;

  constructor(
    nombre: string,
    apellido1: string,
    apellido2: string,
    username: string,
    password: string,
    guid: string,
    email: string,
    telefono: string,
    admin: boolean,
  ) {
    this.nombre = nombre;
    this.apellido1 = apellido1;
    this.apellido2 = apellido2;
    this.username = username;
    this.password = password;
    this.guid = guid;
    this.email = email;
    this.telefono = telefono;
    this.admin = admin;
  }
}

File: src/Services/Users/Users.Service.Queries/DTOs/UsuarioDto.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Users.Service.Queries.DTOs
{
    public class UsuarioDto
    {
        public int id { get; set; }
        public string nombre { get; set; }
        public string apellido1 { get; set;}
        public string apellido2 { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        public Guid guid { get; set; }
        public string email { get; set; }
        public string telefono { get; set; }
        public bool admin { get; set; }
        public bool registering { get; set; }
        public string otp { get; set; }
    }
}
